rollup.config.js

```diff
@@ -3,13 +3,27 @@ import peerDepsExternal from 'rollup-plugin-peer-deps-external'
 import { nodeResolve } from '@rollup/plugin-node-resolve'
 import url from '@rollup/plugin-url'
 import commonjs from '@rollup/plugin-commonjs'
+import alias from '@rollup/plugin-alias'
 import babel from '@rollup/plugin-babel'
 import { visualizer } from 'rollup-plugin-visualizer'
 
 import packageJson from './package.json'
 
 const extensions = DEFAULT_EXTENSIONS.concat(['.ts', '.tsx'])
 
+const aliasPlugin = alias({
+  entries: [
+    { find: 'Components/*', replacement: './src/components/*' },
+    { find: 'Constants/*', replacement: './src/constants/*' },
+    { find: 'Foundation/*', replacement: './src/foundation/*' },
+    { find: 'Hooks/*', replacement: './src/hooks/*' },
+    { find: 'Layout/*', replacement: './src/layout/*' },
+    { find: 'Types/*', replacement: './src/types/*' },
+    { find: 'Utils/*', replacement: './src/utils/*' },
+    { find: 'Worklets/*', replacement: './src/worklets/*' },
+  ],
+})
+
```

Comment:
현재 방식대로라면 rollup 이 올바르게 번들링을 수행하지 못하고 비어있는 package를 생성하게 됩니다. build 했을 때 파일을 확인해주세요~
다음과 같이 바꾸면 문제 없이 잘 됩니다.
```suggestion
const aliasPlugin = alias({
  entries: {
    Components: path.resolve(projectRootDir, 'src', 'components'),
    Constants: path.resolve(projectRootDir, 'src', 'constants'),
    Foundation: path.resolve(projectRootDir, 'src', 'foundation'),
    Hooks: path.resolve(projectRootDir, 'src', 'hooks'),
    Layout: path.resolve(projectRootDir, 'src', 'layout'),
    Types: path.resolve(projectRootDir, 'src', 'types'),
    Utils: path.resolve(projectRootDir, 'src', 'utils'),
    Worklets: path.resolve(projectRootDir, 'src', 'worklets'),
  },
})
```

Reply:
path.resolve는 따로 빼는게 깔끔하겠죵?

Reply:
![image](https://user-images.githubusercontent.com/33861398/145019587-2837f472-e5d7-4884-83d3-907da7f0b7c3.png)
![image](https://user-images.githubusercontent.com/33861398/145019616-9a2a7e97-cccd-440b-b4e3-27ad39f491ef.png)

실제로 aliasing이 잘 적용된 모습

Reply:
```typescript
import path from 'path'
const projectRootDir = path.resolve(__dirname)
```